.github/workflows/ci.yml

```diff
@@ -35,3 +39,5 @@ jobs:
       run: scripts/xcode_select_current_version.sh
     - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
       run: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
+    - script: pod lib lint
```

Comment:
How does the CI know which `cocoapod` version to use for the script?

Reply:
I think it is whatever it has one the virtual machine. similar to Cocoapod publishing :/